replace gpkg method for raster by cog usage (#256)
opengisch/QField-docs: documentation/reference/data-format.en.md

### Raster data

Raster data can become quite big quickly, when working with uncompressed 
tiff files it's often several Gb of data. Especially on mobile devices,
this is inefficient.

### Use COG (Cloud Optimized GeoTIFF)

> COG format will offer best user experience for offline basemap.

Reviewer: ```suggestion
The [Cloud Optimized Geotiff](COG)](https://www.cogeo.org/) format will offer best user experience for offline basemaps.
```

> Combine with JPEG compression, it will reduce the raster size.

Reviewer: ```suggestion
Combined with JPEG compression, it will reduce the raster size.
```

> The following commands will convert a file called `raster.tif` to a COG file `raster_cog.tif` using JPEG compression. Make sure you adjust `EPSG:21781` to your desired CRS and `NUM_THREADS` (for example : 6) if you do not want to use all your computer ressources.

Reviewer: Is it possible to skip the `a_srs` parameter and create an "advanced" example below?

I'd also skip the note on threads or mention it also in an advanced section below.

Author: I've keep it as it was write for GPKG example. But yes, we can simplify this part as I remove also resampling method.
And follow your suggestion and add advanced section for extra parameter : `SRS`, `Num_THREAD` and also `RESAMPLING` method ?

Reviewer: Sample EPSG should be `EPSG:2056` if we want to stick to a swiss CRS (`21781` is outdated).

### Use COG (Cloud Optimized GeoTIFF)

> The [Cloud Optimized Geotiff](https://www.cogeo.org/) format will offer best user experience for offline basemaps.

Reviewer: ```suggestion
The [Cloud Optimized Geotiff (COG)](https://www.cogeo.org/) format will offer best user experience for offline basemaps.
```
